setup.py

```diff
@@ -74,7 +74,7 @@ def run(self):
     "pywin32>=225; sys_platform == 'win32'",
     "networkx>=2.1,<2.4",
     "pydot>=1.2.4",
-    "speedcopy>=2.0.1; sys_platform == 'win32'",
+    "speedcopy>=2.0.1; python_version < '3.8' and sys_platform == 'win32'",
```

Comment:
A lot of dependencies don't have  `classifiers` for `Python3.8`, but seems to have worked.